### PR TITLE
Navbar component

### PR DESCRIPTION
**Pull request by rutu14**
Preview: https://deploy-preview-13--friendly-kilby-44e498.netlify.app/documentation/navbar.html

### CODE REVIEW

File: components/component.css [hunk 1]
```diff
@@ -11,6 +11,7 @@
 @import url(/components/grid.css);
 @import url(/components/modal.css);
 @import url(/components/rating.css);
+@import url(/components/navbar.css);
```
Comment (purvasharmadev) [inline components/component.css:15]: Remove extra spaces.

Comment (rutu14) [reply components/component.css:15]: Hey Purva, the space is for differentiating components and other classes.

**Review by purvasharmadev — APPROVED** [review]
looks good to merge. Nice Work! A descriptive PR will be helpful.

**Comment by rutu14** [comment]
> looks good to merge. Nice Work! A descriptive PR will be helpful.

Thanks, will work on descriptive PRs.

File: components/component.css (hunk 1 above)
Comment (purvasharmadev) [reply components/component.css:15]: Ok, I suggest adding a comment to differentiate.

File: components/navbar.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+.nav-items{
+    margin-left: auto;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.nav-right{
+    margin-right: auto;
```
Comment (sudipkundu999) [inline components/navbar.css]: `.nav-right` items should have a `margin-left:auto` isn't?
or is the naming of class wrong here?

Comment (rutu14) [reply components/navbar.css]: This class is for moving the nav items to right corner. Should I change the name of the class?

**Review by sudipkundu999 — APPROVED** [review]
Good to merge!

**Pull request merged**